testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawITSupport.java

```diff
@@ -35,6 +36,7 @@
 /**
  * Support class for raw ITs.
  */
+@Category(RawTestGroup.class)
```

Comment:
Minor, but to have categories inherited (by having them on *Support classes only) you'd need to bump to junit 4.12.